src/components/tasks/CreateTasks.jsx

```diff
@@ -0,0 +1,49 @@
+import { Fragment, useEffect, useRef, useState } from "react";
+import { useDispatch } from "react-redux";
+import PropTypes from "prop-types";
+import "src/components/tasks/CreateTasks.component.scss";
+import { addTasks } from "src/store/actions/taskActions";
+
+const CreateTasks = ({ showTask, onShow }) => {
+  const dispatch = useDispatch();
+  const [task, setTask] = useState("");
+  const inputRef = useRef(null);
+
+  function focusInputRef() {
+    inputRef.current.focus();
+  }
+
+  useEffect(focusInputRef, []);
+
+  function handleChange(event) {
+    setTask(event.target.value);
+  }
+
+  function handleAddTask() {
+    if (task) {
+      dispatch(addTasks(task));
+      setTask("");
+      onShow(!showTask);
```

Comment:
What is the purpose of `onShow` callback here ? Can we rename it more relatable format which explains the purpose if this callback ?